### ISSUE TITLE
Potential Venues

### ISSUE THREAD

**Issue by shnizzedy** [ChildMindInstitute/HBN-wearable-analysis#6]
- ~~[BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/about)~~ (already submitted and rejected)

> *BMC Bioinformatics* is an open access, peer-reviewed journal  that considers articles on all aspects of the development, testing and novel application of computational and statistical methods for the modeling and analysis of all kinds of biological data, as well as other areas of computational biology.

-----

- [Current Directions in Psychological Science](http://www.psychologicalscience.org/publications/current_directions/instructions-for-authors)

> The major purpose of *Current Directions in Psychological Science* (CD) is to keep readers abreast of developments across the broad spectrum of the behavioral sciences. The journal publishes brief reviews intended to alert a *general audience* to important new developments in research, theory, methods, and applications, in language that is accessible to a much wider audience than the target audience of other scientific journals.

If we're comfortable with a partially closed journal here, plus substantial trimming.
http://www.psychologicalscience.org/publications/current_directions/instructions-for-authors
http://www.psychologicalscience.org/publications/current_directions/cd-submissions

> i don't like the idea of substantial trimming.
— @binarybottle 

-----

- [Psychiatry and Psychology at PeerJ](https://peerj.com/about/aims-and-scope/)

> 1. *PeerJ* is an Open Access, peer-reviewed, scholarly journal. It considers articles in the Biological Sciences, Medical Sciences, and Health Sciences.
> 2. *PeerJ* does not publish in the Physical Sciences, the Mathematical Sciences, the Social Sciences, or the Humanities (except where articles in those areas have clear applicability to the core areas of Biological, Medical or Health sciences).
> 3. *PeerJ* only considers Research Articles and Literature Review Articles. It does not accept Hypothesis Papers, Commentaries, Opinion Pieces, Case Studies, Case Reports etc. which may instead be submitted to [*PeerJ Preprints*](https://peerj.com/about/publications/#PeerJ-Preprints).
> 4. *PeerJ* [evaluates articles](https://peerj.com/about/editorial-criteria) based only on an objective determination of scientific and methodological soundness, not on subjective determinations of 'impact,' 'novelty' or 'interest'.
> 5. *PeerJ* applies the highest standards to everything it does - specifically, the publication places an emphasis on research integrity; high ethical standards; constructive peer-review; exemplary production quality; and leading edge online functionality.
> 6. PeerJ does not accept pre-submission enquiries; however, if you are unsure whether or not your article fits our Scope then please email us at: `editorial.support`@peerj.com.

https://peerj.com/about/aims-and-scope/
https://peerj.com/about/editorial-criteria/

I'm not sure our sample size suffices for the rigor requirement: "The investigation must have been conducted rigorously and to a high technical standard."

> probably not, but may still be worth a try. submit a letter of inquiry?
— @binarybottle 

------

- [PLOS ONE](http://journals.plos.org/plosone/s/journal-information)
> *PLOS ONE* features reports of original research from all disciplines within science and medicine. By not excluding research on the basis of subject area, *PLOS ONE* facilitates the discovery of connections between research whether within or between disciplines. 
> 
> We will also consider the following article types:
>
> - **Systematic reviews.** We consider publishing systematic reviews only if the methods ensure the comprehensive and unbiased sampling of existing literature.
> - **Submissions describing methods, software, databases, or other tools.** We consider submissions describing methods, software, databases, or other tools if they follow the appropriate reporting guidelines.
> - **Qualitative research.** We consider publishing qualitative research only if it adheres to appropriate study design and reporting guidelines.
> - **Studies reporting negative results.**

**Comment by shnizzedy** [comment]
From the Transfer Desk, in my initial rank ordering:

- [Journal of Clinical Monitoring and Computing](http://www.springer.com/medicine/anesthesiology/journal/10877/PS2?detailsPage=aboutThis)

> *... the platform where clinicians and engineers discuss clinical needs and applications and their technical solutions ...*
> 
> - Dedicated to the exchange of knowledge and clinical information on clinical monitoring
> - Includes evaluations of equipment, instrumentation and technology
> - 92% of authors who answered a survey reported that they would definitely publish or probably publish in the journal again
> The *Journal of Clinical Monitoring and Computing* is a clinical journal publishing papers related to technology in the fields of anaesthesia, intensive care medicine, emergency medicine, and peri-operative medicine.
> 
> The journal has links with numerous specialist societies, including editorial board representatives from the European Society for Computing and Technology in Anaesthesia and Intensive Care (ESCTAIC), the Society for Technology in Anesthesia (STA), the Society for Complex Acute Illness (SCAI) and the NAVAt (NAVigating towards your Anaestheisa Targets) group.
> 
> The journal publishes original papers, narrative and systematic reviews, technological notes, letters to the editor, editorial or commentary papers, and policy statements or guidelines from national or international societies.   The journal encourages debate on published papers and technology, including letters commenting on previous publications or technological concerns.  The journal occasionally publishes special issues with technological or clinical themes, or reports and abstracts from scientific meetings. Special issues proposals should be sent to the Editor-in-Chief.  Specific details of types of papers, and the clinical and technological content of papers considered within scope can be found in instructions for authors.

-----
- [Brain Informatics](http://www.springer.com/computer/ai/journal/40708)

> *Brain Informatics* is an international, peer-reviewed, interdisciplinary open-access journal published under the brand SpringerOpen, which provides a unique platform for researchers and practitioners to disseminate original research on computational technologies related to the human brain and cognition. This journal will publish high-quality original research papers, brief reports and critical reviews in all theoretical, technological, clinical and interdisciplinary studies that make up the field of brain informatics and its applications in brain-inspired intelligent systems, health studies, etc.
> 
> The *Brain Informatics* journal addresses the computational, cognitive, physiological, biological, physical, ecological and social perspectives of brain informatics, as well as topics relating to mental health and well-being. It also welcomes emerging information technologies, including but not limited to Internet/Web of Things (IoT/WoT), cloud computing, big data analytics and interactive knowledge discovery related to brain research. The journal also encourages submissions that explore how advanced computing technologies are applied to and make a difference in various large-scale brain studies and their applications.
>
> Informatics-enabled studies are transforming brain science. New methodologies enhance human interpretive powers when dealing with big data sets increasingly derived from advanced neuro-imaging technologies, including fMRI, PET, MEG, EEG and fNIRS, as well as from other sources like eye-tracking and from wearable, portable, micro and nano devices. New experimental methods, such as in toto imaging, deep tissue imaging, opto-genetics and dense-electrode recording are generating massive amounts of brain data at very fine spatial and temporal resolutions. These technologies allow measuring, modeling, managing and mining of multiple forms of big brain data. *Brain informatics* techniques for analyzing all the data will help achieve a better understanding of human thought, memory, learning, decision-making, emotion, consciousness and social behaviors. These methods also assist in building brain-inspired, human-level wisdom-computing paradigms and technologies, improving the treatment efficacy of mental health and brain disorders.
>
>The publication costs for *Brain Informatics* are covered by the journal so authors do not need to pay an article-processing charge.
>
> The journal operates a single-blind peer-review system, where the reviewers are aware of the names and affiliations of the authors, but the reviewer reports provided to authors are anonymous.
> 
> All articles published are made freely and permanently accessible online immediately upon publication, without subscription charges or registration barriers. Further information about open access can be found at https://www.springeropen.com/about/open-access.
> 
> As authors of articles published in *Brain Informatics* you are the copyright holders of your article and have granted to any third party, in advance and in perpetuity, the right to use, reproduce or disseminate your article, according to the SpringerOpen copyright and license agreement (https://www.springeropen.com/get-published/copyright/copyright-and-license-agreement).
>
> For those of you who are US government employees or are prevented from being copyright holders for similar reasons, SpringerOpen can accommodate non-standard copyright lines. Please contact info@springeropen.com if further information is needed.

-----
- [Multimedia Tools and Applications](http://www.springer.com/computer/information+systems+and+applications/journal/11042)

> - First journal in the field, well recognized in the multimedia community
> - Editorial Board composed of the world's foremost experts on multimedia
> - Theoretical as well as application oriented
> - Publishes original research articles on multimedia development and system support tools as well as case studies of multimedia applications
Features experimental and survey articles
> - 93% of authors who answered a survey reported that they would definitely publish or probably publish in the journal again
> *Multimedia Tools and Applications* publishes original research articles on multimedia development and system support tools as well as case studies of multimedia applications. It also features experimental and survey articles. The journal is intended for academics, practitioners, scientists and engineers who are involved in multimedia system research, design and applications. All papers are peer reviewed.
> Specific areas of interest include:
> 
> **- Multimedia Tools:**
> 
> **- Multimedia Applications:**
> 
> **- Prototype multimedia systems and platforms**
> 
> **- Home**
> 
> **- Education and Training**
> 
> **- Operations**
> 
> **- Public**
>
> **- Business Office**
>
> **- Visual Information Systems**

-----

- [Scientific Reports](https://www.nature.com/srep/publish/reasons-to-publish)

> *Scientific Reports* covers all areas of the natural and clinical sciences. Open access and continuous online publication enables articles to be published swiftly, ready to be accessed by anyone, anywhere, at any time. With over a million article page views per month, researchers can be confident that they’re maximising the reach and discoverability of their paper.

-----

- [Behavioral and Brain Functions](https://behavioralandbrainfunctions.biomedcentral.com/about)

> *Behavioral and Brain Functions* (*BBF*) is an open-access journal that welcomes manuscripts that provide insight into behavior and brain function or dysfunction, giving priority to manuscripts that combine both neurobiology and behavior.

-----

- [Journal of Medical Systems](http://www.springer.com/public+health/journal/10916)

> - Covers new systems and methods in medical office administration; pathology, radiology, and pharmaceutical delivery systems; medical records management and more
Offers articles, essays, and studies across the entire spectrum of medical systems
> - Includes a special section devoted to status reports on current installations
> - Fast online publication: most articles are published within 2 months after acceptance
> - 100% of authors who answered a survey reported that they would definitely publish or probably publish in the journal again
> The *Journal of Medical Systems* provides a forum for the presentation and discussion of the increasingly extensive applications of new information systems techniques and methods across all health care settings. It features six sections focusing on:
>
> - Mobile & Wireless Health
> - Quality Improvement
> - Transaction Processing Systems
> - Image & Signal Processing
> - Patient Facing Systems
> - Education & Training
> Journal publishes informative original articles, reviews, and studies across the entire scale of medical systems, from large hospital programs to novel small-scale medical services. Education is an integral part of this amalgamation of sciences, and selected articles are published in this area.

-----

- [BMC Public Health](https://bmcpublichealth.biomedcentral.com/about)

> *BMC Public Health* is an open access, peer-reviewed journal that considers articles on the epidemiology of disease and the understanding of all aspects of public health. The journal has a special focus on the social determinants of health, the environmental, behavioral, and occupational correlates of health and disease, and the impact of health policies, practices and interventions on the community.

**Comment by binarybottle** [comment]
It looked like the Journal of Clinical Monitoring and Computing would be perfect, but when they say "monitoring" they mean something too specific: "The Journal of Clinical Monitoring and Computing is a clinical journal publishing papers related to technology in the fields of anaesthesia, intensive care medicine, emergency medicine, and peri-operative medicine."

I would check PeerJ: "PeerJ does not accept pre-submission enquiries; however, if you are unsure whether or not your article fits our Scope then please email us at: editorial.support@peerj.com."

Of these, my choices for most appropriate would be BMC Bioinformatics and PLoS ONE.

**Comment by shnizzedy** [comment]
I'm sorry, I should have been clearer in my question.

*BMC Bioinformatics* rejected the paper as out of scope but recommended it to the Springer Transfer Desk, which in turn recommended [the following journals](https://github.com/ChildMindInstitute/HBN_wearable_evaluation/issues/6#issuecomment-335209267):

- *Journal of Clinical Monitoring and Computing*
- *Brain Informatics*
- *Multimedia Tools and Applications*
- *Scientific Reports*
- *Behavioral and Brain Functions*
- *Journal of Medical Systems*
- *BMC Public Health*

Using the Transfer Desk, we can go straight to the seen-by-an-editor stage at any or all of these journals. The order above is my ranked preference to try them in (since the rules dictate the article can only be under consideration by one journal at a time). My question was 
1. Does this rank order seem okay to you?

I still have that question, but now I have additional questions.

If/when we submit to *PLoS ONE*, we'll have to withdraw the submission that's already passed muster for formatting and length. I'm inclined to cycle through the publisher-recommended journals before starting the process over. 

2. Do you think it would be better to withdraw the submission, submit to *PLoS ONE*, and only then, if rejected, submit anew to one of these rather than rolling through them first?

3. Do you think this work is too far afield to submit to *Journal of Clinical Monitoring and Computing* at all?

My feeling re: 3. is that the article would almost certainly need some reframing to emphasize its clinical import, but that the work is not completely irrelevant to their scope.

Re: 2., my inclination is to accept the offer to proceed directly to these editors.

-----

To reiterate my ranking is as follows:

- ~~*BMC Bioinformatics*~~ (already submitted and rejected)
- Transfer Desk
  - *Journal of Clinical Monitoring and Computing*
  - *Brain Informatics*
  - *Multimedia Tools and Applications*
  - *Scientific Reports*
  - *Behavioral and Brain Functions*
  - *Journal of Medical Systems*
  - *BMC Public Health*
- *PLoS ONE*
- *Current Directions in Psychological Science*
- *Psychiatry and Psychology at PeerJ*

**Comment by binarybottle** [comment]
It seems out of scope for JoCMaC, though this acronym is pretty neat.  If you'd like you can contact the editor of JoCMaC, but the open access articles I saw were in line with what they say they are interested in, which is not what we're interested in.  That list of journals does not reflect our intended readership -- they only recommended them because Springer stands to gain money.  I say we go for PLoS ONE unless JoCMaC turns out to be a potentially good home.

**Comment by shnizzedy** [comment]
Alright. I'll reach out re: *JoCMaC* and from there withdraw and submit at *PLoS ONE*.

**Comment by binarybottle** [comment]
Alternatively, you could start a new JoCMaC competitor journal for "Jon's only Child Mind acceptance Collective".

**Comment by shnizzedy** [comment]
"**Jo**n **C**lucas: **M**ind **a**t **C**hild"?

**Comment by binarybottle** [comment]
"Mind at Child" -- I think we've got a new name for the institute -- "Institut de Mind at Child"!

**Comment by shnizzedy** [comment]
#### Current status & ranking:

- ~~*BMC Bioinformatics*~~ (submitted and rejected)
- [Transfer Desk](http://www.editorialmanager.com/desk/default.aspx)
  - [_**Journal of Clinical Monitoring and Computing**_](http://www.springer.com/medicine/anesthesiology/journal/10877/PS2?detailsPage=aboutThis) (consulting editor(s) re: scope)
- *PLoS ONE*
- *Current Directions in Psychological Science*
- *Brain Informatics*
- *Multimedia Tools and Applications*
- *Scientific Reports*
- *Behavioral and Brain Functions*
- *Journal of Medical Systems*
- *BMC Public Health*
- *Psychiatry and Psychology at PeerJ*

**Comment by shnizzedy** [comment]
#### Current status & ranking:

- ~~*BMC Bioinformatics*~~ (submitted and rejected)
- ~~Transfer Desk~~
  - ~~*Journal of Clinical Monitoring and Computing*~~
(
      > The editor comments were: "I do not believe this manuscript to be within the scope of the Journal of Clinical Monitoring and Computing. The clinical area of interest does not directly cover those of anaesthesia, intensive care or peri-operative medicine, those within the scope of the journal."
      >
      > In light of the editor comments I have also added another 3 journals for your consideration. The other journals suggested for your manuscript were:
      > 
      > * BMC Annesthesiology [sic][ . . . ]
      > * Journal of Intensive Care[ . . . ]
      > * Perioperative Medicine

    — 🤖 ?

    )
- [__*PLoS ONE*__](http://journals.plos.org/plosone/) ([in-house technical check](http://pone.edmgr.com/))
- *Current Directions in Psychological Science*
- *Brain Informatics*
- *Multimedia Tools and Applications*
- *Scientific Reports*
- *Behavioral and Brain Functions*
- *Journal of Medical Systems*
- *BMC Public Health*
- *Psychiatry and Psychology at PeerJ*

**Comment by shnizzedy** [comment]
[__*PLoS ONE*__](http://journals.plos.org/plosone/)
> progressing to assessment by an in-house editor for checks related to our editorial policies

**Comment by shnizzedy** [comment]
Submitted to [Cognitive Science 2018](http://www.cognitivesciencesociety.org/conference/cogsci-2018/) — July 25th – 28th, 2018, Monona Terrace Community and Convention Center, Madison, WI (December 4)

-----

~~PLoS ONE~~ (November 2):

> PLOS ONE requires all research involving human participants to have been performed in accordance with the Declaration of Helsinki and to have been approved by an appropriate ethics committee in advance of the study,even when the subjects are the researchers themselves. Alternatively, we would expect a relevant ethical body to have confirmed that the need for approval is not necessary. Please note that we do not accept retrospective approval.
> 
> In the future, we strongly recommend that you consult with your research ethics committee before initiating any research involving human participants, regardless of a perceived low risk.
> 
> I am sorry that we cannot be more positive on this occasion, but we hope that you appreciate the reasons for this decision.
